src/v2/es-utils.js

```diff
@@ -54,6 +54,37 @@ class ESClass {
       });
   }
 
+  async healthy(timeout, status) {
```

Comment:
I can't tell what this function does based on the name.

Reply:
I can't tell what format `timeout` is supposed to be or what inputs are allowed (eg: `timeoutMs`?)

Reply:
Re: the function name, see ^^^
THe timeout is a great suggestion! I updated it and also set the defaults in the function parameters.